feeds/oregon-gtfs.com.dmfr.json

```diff
@@ -1413,6 +1266,84 @@
           ]
         }
       ]
+    },
+    {
+      "spec": "gtfs",
+      "id": "f-point~or-us",
+      "urls": {
+        "static_current": "http://oregon-gtfs.trilliumtransit.com/gtfs_data/point-or-us/point-or-us.zip"
+      }
```

Comment:
Please also add an `operators` property with a single record for the new POINT operator. The operator record can also have a `supersedes_ids` property with an array of the old operator Onestop IDs.

If you have any questions about this, feel free to let us know — I know it's not yet well documented.